src/style.css

```diff
@@ -1,3 +1,52 @@
 body {
   margin: 0;
+  display: block;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.link {
+  text-decoration: none;
+  color: black;
+  cursor: pointer;
+  padding: 20px 0 ;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.logo {
+  margin-top: 10px;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__item {
+  margin-left: 20px;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  display: flex;
+}
+
+.is-active:hover {
+  color: #00acdc;
+}
+
+.apple {
+  text-decoration: underline;
+  text-decoration-color: #00acdc;
+  text-underline-offset: 18.5px;
+  text-decoration-thickness: 4px;
+  color: #00acdc;
 }
```

Comment:
First of all that bad name for class.
Second it should be adaptive and responsive, so if you use this class for another link it should be 100% width at all situations and have position at end of block